File: study/this_is_coding_interview/sort/change_ele.js
const input = require("fs")
  .readFileSync("./input.txt", "utf-8")
  .toString()
  .trim()
  .split("\n");
const [N, K] = input.shift().split(" ").map(Number);
const A = input
  .shift()
  .split(" ")
  .map(Number)
  .sort((a, b) => a - b);
const B = input
  .shift()
  .split(" ")
  .map(Number)
  .sort((a, b) => b - a);
let ans = 0;
console.log(A);
console.log(B);
for (let i = 0; i < N; i++) i < K ? (ans += B[i]) : (ans += A[i]);
console.log(ans);

File: study/this_is_coding_interview/sort/change_ele.py

N,K = map(int,input().split())

A = list(map(int,input().split()))
B = list(map(int,input().split()))

A.sort()
B.sort(reverse=True)
print(A,B)
ans = 0
for i in range(len(A)):
  if i<K:
    ans += B[i]
  else:
    ans += A[i]

print(ans)